frontend/src/Page/index.js

```diff
@@ -133,6 +133,21 @@ function Page(props) {
     authors = <Meta title={title}>{authorLinks}</Meta>;
   }
 
+  let custom = [];
+
+  for (let title in props.custom) {
+    // Display only one list item
+    let listContent = props.custom[title];
+
+    // Display more than one list item
+    if (typeof(listContent) == 'object') {
```

Comment:
Please use strict comparison `===` as the default.
```suggestion
    if (typeof(listContent) === 'object') {
```

Reply:
Not sure if checking if this is an object is the right thing to do here – shouldn’t we use `Array.isArray(obj)` in order to check if it is an array?

Reply:
Thanks for catching yes, this is indeed the better approach.